battle/README.md

```diff
@@ -0,0 +1,35 @@
+# Max Battle - developer setup
+
+## Checklist for initial setup
+
+1) Installed git
```

Comment:
This is good and pretty close to what I was going for. The main reason for documenting things is to have you get in the practice of having to describe to others what you did and why when you're communicating about open source. Slightly more detail here (not much) would be good to include. For example, how did you install Git? Did you use a package manager? Did you manually install it? Did you build it from source? What distribution method did you get it from?

Reply:
Ah that makes sense, thanks. I updated the checklist with more detailed notes for some of the steps now.